test/end-to-end/paths/caveat.e2e.js

```diff
@@ -33,12 +25,12 @@ languages.forEach(language => {
         await I.seeSummaryPage(language);
         await I.seePaymentBreakdownPage(language);
         if (testConfigurator.getUseGovPay() === 'true') {
-            await I.seeGovUkPaymentPage();
+            await I.seeGovUkPaymentPage(language);
             await I.seeGovUkConfirmPage();
         }
 
-        const isCaseIDGenerated = await I.checkElementExist('//h1[contains(text(), \'Application complete\')]');
-        if (isCaseIDGenerated) {
+        const caseIDGenerated = await I.checkElementExist('//*[@id="main-content"]/div/div/div[1]/h1');
+        if (caseIDGenerated) {
             I.seeThankYouPage();
         }
```

Comment:
do we need an await here (seeThankYouPage)?

Reply:
seeThankYouPage() method is not an Async method so therefore we don’t need await